tests all fail

```
core/nodereport (master *% u=) % npm test

> nodereport@0.1.0 test /home/sam/w/core/nodereport
> node test/autorun.js

autorun.js: running child process #1 to produce NodeReport on API call
autorun.js: running child process #2 to produce NodeReport on exception
autorun.js: running child process #3 to produce NodeReport on fatal error
autorun.js: running child process #4 to produce NodeReport on SIGUSR2
autorun.js: test results: 
        test 1 :  fail
        test 2 :  fail
        test 3 :  fail
        test 4 :  fail

assert.js:85
  throw new assert.AssertionError({
  ^
AssertionError: false == true
    at ChildProcess.<anonymous> (/home/sam/w/core/nodereport/test/autorun.js:27:10)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)
npm ERR! Test failed.  See above for more details.

```

```
core/nodereport (master u=) % git co -t remotes/rnchamberlain/avoid-jsstacktrace-in-gc
Branch avoid-jsstacktrace-in-gc set up to track remote branch avoid-jsstacktrace-in-gc from rnchamberlain.
Switched to a new branch 'avoid-jsstacktrace-in-gc'
core/nodereport (avoid-jsstacktrace-in-gc u=) % npm i

> nodereport@1.0.4 install /Users/sam/w/core/nodereport
> node-gyp rebuild

gyp WARN download NVM_NODEJS_ORG_MIRROR is deprecated and will be removed in node-gyp v4, please use NODEJS_ORG_MIRROR
gyp WARN download NVM_NODEJS_ORG_MIRROR is deprecated and will be removed in node-gyp v4, please use NODEJS_ORG_MIRROR
gyp WARN download NVM_NODEJS_ORG_MIRROR is deprecated and will be removed in node-gyp v4, please use NODEJS_ORG_MIRROR
  CXX(target) Release/obj.target/nodereport/src/node_report.o
  CXX(target) Release/obj.target/nodereport/src/module.o
  SOLINK_MODULE(target) Release/nodereport.node
clang: warning: libstdc++ is deprecated; move to libc++ with a minimum deployment target of OS X 10.9
  COPY /Users/sam/w/core/nodereport/nodereport.node
  TOUCH Release/obj.target/install.stamp
nodereport@1.0.4 /Users/sam/w/core/nodereport
└── nan@2.4.0 

core/nodereport (avoid-jsstacktrace-in-gc u=) % npm test

> nodereport@1.0.4 test /Users/sam/w/core/nodereport
> node test/autorun.js

autorun.js: running child process #1 to produce NodeReport on API call
autorun.js: running child process #2 to produce NodeReport on exception
autorun.js: running child process #3 to produce NodeReport on fatal error
autorun.js: running child process #4 to produce NodeReport on SIGUSR2
autorun.js: test results: 
        test 1 :  fail
        test 2 :  fail
        test 3 :  fail
        test 4 :  fail

assert.js:85
  throw new assert.AssertionError({
  ^
AssertionError: false == true
    at ChildProcess.<anonymous> (/Users/sam/w/core/nodereport/test/autorun.js:27:10)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)
npm ERR! Test failed.  See above for more details.
```

that's OS X, the original test was on Linux, I'll try Linux when I get home

--- a/.gitignore
+++ b/.gitignore
@@ -41,3 +41,4 @@
 NodeReport*.txt
 npm-debug.log
 nodereport-*.tgz
+nodereport_test.log

--- a/test/test_1.js
+++ b/test/test_1.js
@@ -1,5 +1,5 @@
 // NodeReport API example
-var nodereport = require('nodereport');
+var nodereport = require('../');
 
 console.log('api_call.js: triggering a NodeReport via API call...');
 

--- a/test/test_2.js
+++ b/test/test_2.js
@@ -1,5 +1,5 @@
 // Testcase to produce an uncaught exception
-require('nodereport').setEvents("exception");
+require('../').setEvents("exception");
 
 console.log('exception.js: throwing an uncaught user exception....');
 

--- a/test/test_3.js
+++ b/test/test_3.js
@@ -1,5 +1,5 @@
 // Testcase to produce a fatal error (javascript heap OOM)
-require('nodereport').setEvents("fatalerror");
+require('../').setEvents("fatalerror");
 
 console.log('fatalerror.js: allocating excessive javascript heap memory....');
 var list = [];

--- a/test/test_4.js
+++ b/test/test_4.js
@@ -1,5 +1,5 @@
 // Testcase to loop in Javascript code
-require('nodereport').setEvents("signal");
+require('../').setEvents("signal");
 
 console.log('loop.js: going into loop now.... use kill -USR2 <pid> to trigger NodeReport');
 

